x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -34,12 +36,15 @@ const CompliancePage = () => {
   );
 };
 
-export const ComplianceDashboard = () => (
-  <CspPageTemplate
-    pageHeader={{
-      pageTitle: 'Compliance',
-    }}
-  >
-    <CompliancePage />
-  </CspPageTemplate>
-);
+export const ComplianceDashboard = () => {
+  useCspBreadcrumbs([allNavigationItems.dashboard]);
+  return (
+    <CspPageTemplate
```

Comment:
```suggestion
export const ComplianceDashboard = () => {
  useCspBreadcrumbs([allNavigationItems.dashboard]);
  
  return (
    <CspPageTemplate
```

just added a space, I have a problem with spaces.

Reply:
watch out @ari-aviran , it's contagious. i like them spaces now too 😆